doc/mgmt/SONiC_OC_FDB_get_support_HLD.md

```diff
@@ -0,0 +1,271 @@
+### 3.3.2 Other Process
+## 3.4 SyncD
+## 3.5 SAI
+## 3.6 User Interface
+### 3.6.1 Data Models
+The following open config YANG model is used to implement get support for FDB entries.
+[https://github.com/openconfig/public/blob/master/release/models/network-instance/openconfig-network-instance-l2.yang#L292)
+```
+module: openconfig-network-instance
+    +--rw network-instances
+       +--rw network-instance* [name]
+          +--rw name                       -> ../config/name
+          +--rw fdb
+          |  +--rw config
+          |  |  +--rw mac-learning?      boolean
+          |  |  +--rw mac-aging-time?    uint16
+          |  |  +--rw maximum-entries?   uint16
+          |  +--ro state
+          |  |  +--ro mac-learning?      boolean
+          |  |  +--ro mac-aging-time?    uint16
+          |  |  +--ro maximum-entries?   uint16
+          |  +--rw mac-table
+          |     +--rw entries
+          |        +--rw entry* [mac-address vlan]
+          |           +--rw mac-address    -> ../config/mac-address
+          |           +--rw vlan           -> ../config/vlan
+          |           +--rw config
+          |           |  +--rw mac-address?   yang:mac-address
+          |           |  +--rw vlan?          -> ../../../../../../vlans/vlan/config/vlan-id
+          |           +--ro state
+          |           |  +--ro mac-address?   yang:mac-address
+          |           |  +--ro vlan?          -> ../../../../../../vlans/vlan/config/vlan-id
+          |           |  +--ro age?           uint64
+          |           |  +--ro entry-type?    enumeration
+          |           +--rw interface
+          |              +--rw interface-ref
+          |                 +--rw config
+          |                 |  +--rw interface?      -> /oc-if:interfaces/interface/name
+          |                 |  +--rw subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
+          |                 +--ro state
+          |                    +--ro interface?      -> /oc-if:interfaces/interface/name
+          |                    +--ro subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
+```
+### 3.6.2 CLI
+#### 3.6.2.1 Configuration Commands
+#### 3.6.2.2 Show Commands
+The following CLI commands dump the output of internal FDB entries from STATE_DB with various options (filters), for example, filter based on MAC address, entry type (static/dynamic), VLAN interface, physical/port-channel interfaces and MAC address table count ..etc.
+##### 3.6.2.2.1 show mac address-table
+Syntax
+
+show mac address-table [address ``<mac>`` | count | dynamic { address ``<mac>`` | vlan ``<id>`` | interface { Ethernet ``<id>`` | Port-channel ``<id>`` }} | interface { Ethernet ``<id>`` | Port-channel ``<id>`` } | static { address ``<mac>`` | vlan ``<id>`` | interface { Ethernet ``<id>`` | Port-channel ``<id>`` }} | vlan ``<id>`` }]
```

Comment:
I think this would be more readable if you broke it down into more sections, like: 
3.6.2.2.1 `show mac address-table address`
3.6.2.2.2 `show mac address-table count`
3.6.2.2.3 `show mac address-table dynamic {address | interface}`
3.6.2.2.4 `show mac address-table interface`
3.6.2.2.5 `show mac address-table static`

Reply:
done